cityofaustin/atd-airflow
# Removes webhook and weather DAG and adds Docker CI

This PR:
- Removes the weather DAG
- Removes the webhook stack
- Adds docker build/push CI to our custom airflow image
- Updates the docker compose to use the airflow image from dockerhub


## Steps to test
1. `docker compose up` should get you going. Our airflow image will pull from the hub on your first go.
1. Visit `localhost:8080` to access the airflow UI. Observe that all DAGs have imported without error
2. Visit `localhost:8081` to view the Flower UI. You should see one worker available.
